sdk/typescript/esc/raw/common.ts

```diff
@@ -82,7 +47,7 @@ export const setOAuthToObject = async function (object: any, name: string, scope
         const localVarAccessTokenValue = typeof configuration.accessToken === 'function'
             ? await configuration.accessToken(name, scopes)
             : await configuration.accessToken;
-        object["Authorization"] = "Bearer " + localVarAccessTokenValue;
+        object["Authorization"] = configuration.tokenPrefix + localVarAccessTokenValue;
```

Comment:
Was `Bearer ` wrong?  Looks like this is now using `token `?  Also - is this a space missing?